src/main/java/org/sunbird/cbp/service/CbPlanServiceImpl.java

```diff
@@ -509,6 +509,37 @@ public SBApiResponse getCBPlanListForUser(String userOrgId, String authUserToken
         return response;
     }
 
+    public SBApiResponse requestCbplanConten(SunbirdApiRequest request, String userOrgId, String token){
+        SBApiResponse response = ProjectUtil.createDefaultResponse(Constants.CBP_PLAN_CONTENT_REQUEST_API);
+        Map<String, Object> contentRequest = (Map<String, Object>) request.getRequest();
+        List<Map<String, Object>> competency = (List<Map<String, Object>>) contentRequest.get(Constants.COMPETENCY);
+        List<String> providersOrgId = (List<String>) contentRequest.get("providerList");
+        String description = (String) contentRequest.get(Constants.DESCRIPTION);
+        try{
+            String userId = validateAuthTokenAndFetchUserId(token);
+            Map<String, String> mdoInfo = userUtilityService.getUserDetails(Collections.singletonList(userId), new ArrayList<>()).get(userId);
+            String mdoOrgId = mdoInfo.get(Constants.ROOT_ORG_ID);
+            String competencyJsonString = mapper.writeValueAsString(competency);
+
+            Map<String, Object> propertiesMap = new HashMap<>();
+            propertiesMap.put(Constants.ORG_ID, mdoOrgId);
+            propertiesMap.put(Constants.ID, UUID.randomUUID().toString());
+            propertiesMap.put(Constants.COMPETENCY_INFO , Collections.singletonList(competencyJsonString));
```

Comment:
can't we simply use competency list object ?

Reply:
This is updated, we are storing value as an object.